### ISSUE TITLE
Labor-Preise und Kontrolle

### ISSUE THREAD

**Issue by chrbusch** [chrbusch/GTM-V#416]
NiF wünscht die Erfassung der Preise von Laboruntersuchungen, um damit die von den Laboren eingehenden Kreditorrechnungen automatisiert prüfen zu können.
Bei den "Adressendetails" gibt es einen neue Reiter "Laborpreise".
Darin sieht man - wenn als Funktion u.a. "Labor" angekreuzt ist - eine Eingabeliste für Preise:
Klicken Sie in der ersten freien Zeile in das erste Felder unter "Parameter".
Geben Sie einen oder zwei Buchstaben des gewünschten Parameternamens ein, etwa   an   für "Anti-CMV", und wählen Sie den Parameter aus.
Nun können Sie in der neuen Zeile einen Preis eingeben.
Bitte für ein wichtiges Labor diese Preise eingeben; falls das Labor eine XLS-Liste mit Preisen liefern kann, können wir diese leicht importieren.

![bildschirmfoto 2018-09-05 um 13 46 56](https://user-images.githubusercontent.com/1241899/45091270-2dc1bf80-b112-11e8-98cb-0a0b477e1bdc.JPG)

**Comment by Toelk-A** [comment]
Kosten sind hinterlegt.

Der Posten 14 HCV-Nat setzt sich aus 4 separat aufgeschlüsselten Positionen zusammen... müssen diese Kosten einzeln abgebildet werden?

![image](https://user-images.githubusercontent.com/22314271/45143792-7fbf1f80-b1bc-11e8-996f-cab0bbd54db4.png)

**Comment by chrbusch** [comment]
Dies bitte mit NiF besprechen.
Ziel ist ja die Kontrollierbarkeit der Kreditorrechnungen, die von den Laboren kommen.
Daher denke ich, wir müssen es in etwa so genau erfassen, wie es auf diesen Rechnungen aufgelistet ist.

**Comment by Toelk-A** [comment]
Generell erfolgt die Abrechnung aus dem Medicus als Sammelrechnung 1x im Monat.... Wenn wir alle Positionen aufteilen müssen benötigen wir für HCV-NAT noch 3 weitere Eingabefelder. Wie kann ich diese anlegen?

**Comment by chrbusch** [comment]
Könnte ich eine Seite so einer Abrechnung bekommen, auf der alle Positionen zu sehen sind?
Ich würde dann Ihnen per TeamViewer zeigen, wie Sie selbst fehlende Posten ergänzen könne - auch im Hinblick auf andere Labore.

**Comment by chrbusch** [comment]
Wichtig noch: Was genau beutet "Monat", etwa beim August: Ist das "Datum Befund" im August, das Datum des Untersuchungsauftrages oder was ganz genau legt fest, ob ein Befund abrechnungstechnisch zum August gehört?

**Comment by Toelk-A** [comment]
Wer arbeitet denn nun an diesem Ticket Herr Prüter oder Dr. Busch?

**Comment by chrbusch** [comment]
Bitte eine Seite einer typischen Laborrechnung hier einstellen

**Comment by chrbusch** [comment]
"Der Posten 14 HCV-Nat setzt sich aus 4 separat aufgeschlüsselten Positionen zusammen"

Welche sind das denn? Ist es nicht die eine Zeile in diesem Bild?

![bildschirmfoto 2018-09-13 um 07 31 09](https://user-images.githubusercontent.com/1241899/45468840-1775be00-b727-11e8-9097-d5850e5cbae3.JPG)

**Comment by Toelk-A** [comment]
Nein, das brauchen Sie nichht es befindet sich alles auf der Rechnung:

![image](https://user-images.githubusercontent.com/22314271/45469198-d54d7c00-b728-11e8-8e80-633de2b2fe17.png)

**Comment by chrbusch** [comment]
Labor hat sofort geantwortet; es wird dort nun geprüft, ob wir die Rng digital bekommen können

**Comment by Toelk-A** [comment]
super!

**Comment by Toelk-A** [comment]
müssen nun die Einzelpositionen für HCV aufgelistet werden oder ist das damit erledigt?

**Comment by chrbusch** [comment]
...es ist noch viel besser: Sie müssen nichts mehr ändern; wir bekommen bald die Monatsrechnungen digital eingespielt, was wenig Arbeit macht, weil derselbe Versandweg benutzt wird wie für die Labordaten (LDT).
Sobald Medicus das liefern kann, passen wir nur das Importscript an und dann sollten alle Laborkosten transparent neben den Unters,.Aufträgen stehen.
Der Kollege bei Medicus ist gerade in urlaub, danach geht es hier weiter.

**Comment by chrbusch** [comment]
Maik, bitte dieses Teilprojekt übernehmen.
Montag tel.!
